### DISCUSSION TITLE
System Info Display and RStudio Quarto Path Issues

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#70] [Issues]
Description:
I am experiencing two issues with my current setup on Dat Linux OS:

System Info Display Issue:
The system information looks strange (see attached Picture 1). It seems off compared to the usual output, and I suspect something might be misconfigured.

RStudio Quarto Path Problem:
RStudio is consistently losing track of the Quarto path, which prevents me from creating new Quarto documents. However, existing Quarto documents render without issues.

Unable to Open Existing Project Files:
I am encountering an error (see Picture 2) whenever I try to open existing project files. This is preventing me from continuing my work on existing projects.

![screen2](https://github.com/user-attachments/assets/93c65712-a05e-4afc-bef0-f59ac36ad09d)
![screen](https://github.com/user-attachments/assets/fec4ad83-517e-4473-ad70-dfdd8d1ab073)

**Comment by dat-linux** [comment]
Hi.

Please update your system to latest control panel (1.0.168). 

Re quarto, please uninstall and reinstall the app to get the latest version. Run `ls -ls /opt/quarto/bin` to confirm location.

Re project open, what are you trying to open and from which application?

**Reply by ghost** [reply]
Thanks for your response, 

How to update the system to the latest control panel (1.0.168) ???

I'm trying to open the project in R, it is an R project file

**Reply by ghost** [reply]
It is fine now, i managed to do it

thanks

**Reply by dat-linux** [reply]
If you want to post details of how to reproduce the RStudio problem you're having, I'll be happy to have a quick look. Usually a project is associated with an **.RProject** file?
